### ISSUE TITLE
Improve view granularity

### ISSUE THREAD

**Issue by Fleker** [worktile/ngx-gantt#303]
Good evening,
I added this component to my in-development website, for cooking purposes. While the component works well, its view options are far too broad to work. Even at `GanttViewType.day` my recipe just shows everything together.

Adding additional types like `GanttViewType.minute` would be beneficial for these niche use-cases.

![image](https://user-images.githubusercontent.com/3291635/216732751-42adf601-0416-4de3-bcb6-3f4667be055f.png)

**Comment by walkerkay** [comment]
I think the customized view can solve your problem
for reference example:
- http://gantt.ngnice.com/components/custom-view
- https://github.com/worktile/ngx-gantt/tree/master/example/src/app/gantt-custom-view

Because the minimum unit before is day, the dragging of the minute view may be affected, and we may improve this in the future

**Comment by Fleker** [comment]
Thanks for your reply.

I have created a custom view, though it looks like `primaryDatePoints` loads and provides a default start date a week of last Sunday and a default end date of next Monday.

I added logging and see that `primaryDatePoints` runs before my log in the constructor which shows the correct start/end dates.

Is there a way to update the chart once the new dates are determined? Right now this adds a lot of empty space.

![image](https://user-images.githubusercontent.com/3291635/219823765-db6d221a-44d1-44c3-be17-3e1835db30f7.png)
